inference-engine/src/mkldnn_plugin/nodes/mkldnn_def_conv_node.cpp

```diff
@@ -1066,12 +1126,20 @@ void MKLDNNDeformableConvolutionNode::execute(mkldnn::stream strm) {
         modulation_strides = getParentEdgeAt(3)->getMemory().GetDescWithType<BlockedMemoryDesc>()->getStrides();
     }
 
-    prepareSamplingWeights(src_strides, offsets, off_strides, modulation, modulation_strides);
-
-    if (def_conv_kernel) {
-        executeOptimized(src, weights, dst, src_strides, dst_strides);
+    if (execPtr) {
+        execPtr->prepareSamplingWeights(src_strides, offsets, off_strides, modulation, modulation_strides, enforceRef);
+        execPtr->exec(src, weights, dst, src_strides, wei_strides, dst_strides);
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/8986#discussion_r770053425

I guess better to separate runtime parameters and parameter that can be obtained on compilation stage.
To reduce args number you can pass descriptors instead passing dims and strides separately.

`exec` should take `src, weights, dst, offsets, modulation` 

Also `prepareSamplingWeights` should be called inside `exec`